.github/workflows/ci.yml

```
@@ -35,159 +33,85 @@ jobs:
-
+        uv pip install -r requirements.in
+        uv pip install pytest
     - name: Test with Pytest
-      run: pytest tests/
-
+      run: uv run pytest tests/
     - name: Record Success
       if: success()
-      run: echo "${{ matrix.python-version }}" > python_version_${{ matrix.python-version }}.txt
-
+      run: echo "${{ matrix.python-version }}" > python_version_${{ matrix.python-version.txt }}
```

actionlint (1.7.12)

[error] receiver of object dereference "txt" must be type of object but got "string"

(expression)